bankbot.lic

```diff
@@ -0,0 +1,229 @@
+# want_script_output
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#bankbot
+=end
+
+custom_require.call(%w(common common-money common-validation drinfomon events))
+
+class Bankbot
+  include DRC
+  include DRCM
+
+  def initialize
+    # TODO: Write transactions to a file
+
+    @settings = get_settings
+    @last_action = Time.now
+    @bankbot_debug = UserVars.bankbot_debug
```

Comment:
You might consider using `UserVars.bankbot_debug` everywhere in the script since this is a long-running script. Checking the UseVar directly (instead of caching it in a local var) would mean you can turn debugging on or off while the script is running, without having to restart it.